### ISSUE TITLE
Recast clones a comment

### ISSUE THREAD

**Issue by nickretallack** [zxbodya/flowts#10]
Found something odd.  Commands all run with `--no-prettier`.

Input:
```
// @flow
import a from 'a';

// b
import b, {type btype} from 'b';
```
Output:
```
import a from 'a';

// b
import b from 'b';

// b
import type { btype } from 'b';
```
With `--no-recast`:
```
import a from 'a'; // b

import b from 'b';
import type { btype } from 'b';
```

**Comment by zxbodya** [comment]
looks like related to copying initial import statement - will look into it

**Comment by zxbodya** [comment]
fixed in 0.9.3
